bin/print_rules.ml

```diff
@@ -29,14 +29,15 @@ let print_rule_makefile ppf (rule : Build_system.Rule.t) =
       ; Action.for_shell rule.action
       ]
   in
+  let eval_pred ~dir:_ _ = [] in
```

Comment:
The issue here is that not all predicates are globs, so it's hard to go back to something that is sensible for makefiles. Do we want some limited support perhaps?

Reply:
I guess a much better approximation would be to expand the glob to all the files that match it.
That's what we do in jenga, anyway. It's not perfect because it won't detect new files appear, but it seems better.

Reply:
Fixed by following your suggestion. I recall Jeremie was excited about improving our makefile output, but it requires some extra plumbing so let's leave it for another day.